main.go

```diff
@@ -1,11 +1,116 @@
 package main
 
 import (
-	"github.com/bitrise-io/go-utils/log"
+	_ "embed"
+	"errors"
+	"fmt"
+	"io/ioutil"
 	"os"
+	"strings"
+	"text/template"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/templateutil"
+	"github.com/bitrise-io/stepman/models"
+	"gopkg.in/yaml.v2"
 )
 
+//go:embed README.md.gotemplate
+var readmeTemplate string
+
+func createBackup() error {
+	err := os.Rename("README.md", "README.md.backup")
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return nil
+		}
+		return fmt.Errorf("failed to rename README.md to README.md.backup, error: %s", err)
+	}
+	return nil
+}
+
+func parseStep() (models.StepModel, error) {
+	fileContents, err := ioutil.ReadFile("step.yml")
+	if err != nil {
+		return models.StepModel{}, fmt.Errorf("failed to open step.yml, error: %s", err)
+	}
+
+	stepConfig := models.StepModel{}
+	if err = yaml.Unmarshal(fileContents, &stepConfig); err != nil {
+		return models.StepModel{}, fmt.Errorf("failed to parse step.yml, error: %s", err)
+	}
+
+	return stepConfig, nil
+}
+
+// clean makes Markdown table–compatible text from the input by replacing line breaks with spaces and escaping special
+// characters.
+func clean(text string) string {
```

Comment:
should not we call it `markdownTableCompatibeString`, or something like that, so that we do not need a comment to explain what this function does?